### PR TITLE
updated dropdown design and added few icons

### LINKED ISSUE #17: update dropdown design

### PR DESCRIPTION
**Pull request by manojkannekanti**
resolves #17

### CODE REVIEW

File: src/dropdown/index.jsx [hunk 1]
```diff
@@ -271,13 +248,15 @@ const Dropdown = (props) => {
...
+        size={props.size || 'M'}
+        background={props.background}
       >
-        <Left>{data.content}</Left>
-        <Right>
-          <Icon position={'right'}>
-            <Arrow rotate={90} />
-          </Icon>
-        </Right>
+        {props.icon && <Icon>{props.icon}</Icon>}
```
Comment (NejcZdovc) [inline src/dropdown/index.jsx:254]: update definition now that we accept icon

Comment (manojkannekanti) [reply src/dropdown/index.jsx:254]: didn't understood, do you mean to remove condition.
If yes, there are some places where we don't have icon (like in global app filter, add a page in personal)

Comment (NejcZdovc) [reply src/dropdown/index.jsx:254]: you need to updated definition file `src/typings/index.d.ts` so that it accepts icon prop in typescript

Comment (manojkannekanti) [reply src/dropdown/index.jsx:254]: okay

Comment (manojkannekanti) [reply src/dropdown/index.jsx:253]: done

**Review by NejcZdovc — APPROVED** [review]

**Pull request merged**